phpunit.xml.dist

```diff
@@ -17,14 +17,19 @@
 >
     <php>
         <ini name="date.timezone" value="America/Lima"/>
-        <env name="GIT_VENDOR_NAME" value="gushphp"/>
 
-        <!-- WARNING These locations are only used for testing, don't use your actual Gush installation! -->
         <!--
-        <env name="GUSH_HOME" value="/PATH/TO/GUSH/HOME"/>
-        <env name="GUSH_LOCAL" value="/PATH/TO/GUSH/LOCAL"/>
-        <env name="GUSH_CACHE_DIR" value="/PATH/TO/GUSH/CACHE"/>
+            Uncomment the "GUSH_USE_FS" env-option to use the real filesystem.
```

Comment:
Whoops :) forgot to change that line, I did change in phpunit.xml (but not the dist :smile: )